src/ImageAPI/Utils/ImageExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using SixLabors.Fonts;
+using SixLabors.ImageSharp;
+using MetaDataAPI.Models.DynamoDb;
+using SixLabors.ImageSharp.Processing;
+
+namespace ImageAPI.Utils;
+
+public static class ImageExtensions
```

Comment:
we can make it none-static, using fluent pattern.
all the function here take the same paraameters